casts/free-podcasts-screencasts-ar.md

```diff
@@ -10,3 +10,5 @@
 * [Nakerah Podcast](https://nakerah.net/podcast) - Nakerah Network (podcast)
 * [null++: بالعربي](https://nullplus.plus) - Mohamed Luay, Ahmad Alfy (podcast)
 * [The Egyptian Guy](https://anchor.fm/refaie) - Mohamed Refaie (podcast)
+* [The Weekly Noob](https://theweeklynoob.netlify.app) - Nabil Tharwat (podcast)
+* [Your Big Brother Metwally](https://anchor.fm/metwally) - Ahmed Metwally (podcast)
```

Comment:
Is this your translation? usually we use the title as given by the resource

Reply:
Ok I will use the original title for this one 
* [Your Big Brother Metwally](https://anchor.fm/metwally) - Ahmed Metwally (podcast)

But for the weekly noob => this original name

Reply:
@eshellman Solved Bro!